ManageRDSessions/ManageRDSessions.psm1

```diff
@@ -67,8 +74,31 @@ function Get-sbRDSession {
         [string[]]$UserName = $null
     )
 
+    Begin {
+        if (-not($ConnectionBroker)) {
+            Write-Verbose "[BEGIN  :] No Connection Broker specified, querying and connecting to local host FQDN."
+            $computerinfo = Get-CimInstance -ClassName Win32_ComputerSystem
+            $ConnectionBroker = "$($computerInfo.DNSHostName).$($computerInfo.Domain)"
+        } else {
```

Comment:
Is this blank `else {}` meant to be here?

Reply:
It is not :|